src/app/error.tsx

```diff
@@ -0,0 +1,34 @@
+'use client'
+
+import Typography from '@/components/ui/Typography'
+import { Button } from '@/components/ui/button'
+import absurd from '../../public/absurd 2.png'
+import Image from 'next/image'
+import { useRouter } from 'next/navigation'
+
+export default function Error({
```

Comment:
why is error passed when not used?

Reply:
It probably remained when I tried to figure out where error.tsx was supposed to be.